Description: Use Debian cohomcalg package instead of cohomcalg shipped with M2.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-05-05

--- a/M2/Macaulay2/packages/CohomCalg.m2
+++ b/M2/Macaulay2/packages/CohomCalg.m2
@@ -10,7 +10,7 @@
         Headline => "interface to CohomCalg software for computing cohomology of torus invariant divisors on a toric variety",
         PackageExports => {"NormalToricVarieties"},
         Configuration => {
-            "executable" => "",
+            "executable" => "cohomcalg",
             "silent" => "false"
             }
         )
--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -971,9 +971,7 @@
      	  # Also, the package CohomCalg looks only in the directory where Macaulay2's binaries are
 	  AC_MSG_RESULT([yes, will build anyway])
           BUILD_cohomcalg=yes
-     else AC_MSG_RESULT([yes, will build anyway])
-     	  # the package CohomCalg looks only in the directory where Macaulay2's binaries are
-          BUILD_cohomcalg=yes
+     else AC_MSG_RESULT([yes])
      fi
 else AC_MSG_RESULT([no, will build])
      BUILD_cohomcalg=yes
